src/lib.rs

```diff
@@ -74,21 +74,28 @@ extern crate error_chain;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde;
-#[macro_use] 
+#[macro_use]
 extern crate serde_json;
+#[macro_use]
+extern crate log;
 
 extern crate handlebars;
 extern crate pulldown_cmark;
 extern crate regex;
 
+#[cfg(test)]
 #[macro_use]
-extern crate log;
+extern crate pretty_assertions;
+#[cfg(test)]
+extern crate tempdir;
```

Comment:
actually I meant tempdir here. pretty_assertions cannot be imported anywhere  else due to macros.